src/ignitus-Internals/index.js

```diff
@@ -24,8 +24,16 @@ export class ErrorBoundary extends React.Component {
     if (errorInfo) {
       return (
         <div className="container">
-          <h2>Something went wrong.</h2>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
+          <h2
+           style={{ textAlign: 'center', marginTop: '4em', color: '#000066' }}>
```

Comment:
Inline styles are still there, please create a seperate `.scss file` and place them there and import it in this component.